config/apis.ts

```diff
@@ -40,16 +51,27 @@ const keyAuthAPIList: KeyAuthAPI[] = [
     key: 'claimsAttributes',
     name: 'Claims Attributes API',
   },
+  {
+    acl: 'hlr',
+    key: 'decisionReviews',
+    name: 'Decision Reviews API',
+  },
   {
     acl: 'va_facilities',
     key: 'facilities',
     name: 'VA Facilities API',
   },
+  {
+    acl: 'loan_guaranty',
+    key: 'loanGuaranty',
+    name: 'Loan Guaranty API',
+  },
   {
     acl: 'va_forms',
     key: 'vaForms',
     name: 'VA Form API',
   },
+
   {
     acl: 'veteran_confirmation',
     key: 'confirmation',
```

Comment:
does Claims Attributes need to be in here? it's still in the frontend even though it's disabled. [asked about this in Slack](https://lighthouseva.slack.com/archives/CAZF880HW/p1629482551074500)